childrenInAccount.php

```diff
@@ -55,13 +55,13 @@
                         <tbody class="standout">';
                         foreach($children as $acct){
                             echo '<tr>';
-                            echo '<td><a href=childAccount.php?id=' . $acct->getID() . '>' . $acct->getID() . '</a></td>';
```

Comment:
I think this should break? There is no getId() function in children.php, only getID()

Reply:
you're right, I'll change it back